packages/core/core/src/services/document-service/transform/relations/extract/data-ids.ts

```diff
@@ -0,0 +1,85 @@
+import { isObject } from 'lodash/fp';
+import { EntityService, Attribute, Common } from '@strapi/types';
+import { traverseEntity } from '@strapi/utils';
+import { IdMap } from '../../id-map';
+import { ShortHand, LongHand, ID } from '../utils/types';
+import { isShortHand, isLongHand } from '../utils/data';
+
+/**
+ *  Get relation ids from primitive representation (id, id[], {id}, {id}[])
+ */
+const handlePrimitive = (
+  relation: ShortHand | LongHand | ShortHand[] | LongHand[] | null | undefined | any
+) => {
+  if (!relation) return []; // null
```

Comment:
I'm surprised we don't have the eslint rule for this 🤔  but can you refactor to open the {} for readability 🙏